src/api/app/views/webui/request/beta_show_tabs/_rpm_lint.html.haml

```diff
@@ -0,0 +1,29 @@
+- if package
+  :ruby
+    ajax_data = {}
+    ajax_data['project'] = h(project) if project
+    ajax_data['package'] = h(package) if package
+    ajax_data['is_staged_request'] = h(is_staged_request) if is_staged_request
```

Comment:
This looks like could be a View Component

Reply:
I don't think it is strictly necessary to do these changes in a View Component in this pull request. But it could be done in a follow-up pull request.

Reply:
Fine for me